assets/js/base/components/product-list/index.js

```diff
@@ -52,6 +52,7 @@ const generateQuery = ( { sortValue, currentPage, attributes } ) => {
 	};
 	return {
 		...getSortArgs( sortValue ),
+		catalog_visibility: 'visible',
```

Comment:
Should it be `catalog` instead of `visible`? I assumed the _All Products_ block had to have feature-parity with the _Shop_ page. But if that's not the case, forget about this comment. :slightly_smiling_face: